presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadataFactory.java

```diff
@@ -124,6 +125,69 @@ public HiveMetadataFactory(
                 hiveFileRenamer);
     }
 
+    public HiveMetadataFactory(
```

Comment:
Add a comment here to clarify the intent of this refactoring.
//Constructor to bridge the changes from java.time to joda for the time-being.